### DISCUSSION TITLE
Any plans for WinUI ?

### DISCUSSION THREAD

**Discussion by Stefan6440** [Taiizor/ReaLTaiizor#137] [General]
I am very glad for the Material Theme, using winforms.  I would like to migrate my application to WinUI 3. I quess other users might already are following this road, or they will later on.
Just wanted to hear if you already are planning ReaLTaiizor WinUI in the future.  I might want to contribute to the development, at least for the Material Controls.

**Comment by Taiizor** [comment]
I'm glad you're satisfied. The project doesn't have an active development process. I've been trying to keep it updated with minor changes for a long time. So, I have no intention of supporting this project for WPF or WinUI. There are already much better UI libraries for WPF or WinUI. You can turn to them and take your project to a higher level.
